redirects.csv

```diff
@@ -197,3 +197,5 @@ https://discipleship.crossroads.net/*,https://www.crossroads.net/huddle,301!,mas
 /getstarted/thank-you,https://online.crossroads.net/,301!
 /live,https://online.crossroads.net/,301!
 /*,/404.html,404
+/signin,/https://online-int.crossroads.net,301 Cookie=fromOnline
```

Comment:
What does `Cookie=fromOnline` do? Also, this file is processed in order so having this rule below the wildcard rule `/*` will prevent this new addition from ever getting evaluated. I'd recommend moving it up one line.

Reply:
It checks for the presence of the cookie "fromOnilne" and redirects if it is present. Info (Here)[https://docs.netlify.com/routing/redirects/redirect-options/#redirect-by-cookie-presence]

Also good to know or the evaluation order.